tests/model/test_rf_model.py

```
@@ -1,80 +1,89 @@
 import numpy as np
+import pandas as pd
 import pytest
 from sklearn.compose import ColumnTransformer
```

Ruff

`sklearn.compose.ColumnTransformer` imported but unused

Remove unused import: `sklearn.compose.ColumnTransformer`

(F401)